datashader/transfer_functions.py

```diff
@@ -116,7 +152,7 @@ def interpolate(agg, low=None, high=None, cmap=None, how='cbrt'):
         agg = agg.where(agg > 0)
         offset = agg.min()
     how = _normalize_interpolate_how(how)
-    data = how(agg - offset)
+    data = how(agg.data - offset.data)
```

Comment:
But there's no corresponding change in the functions themselves in this PR, right?  So is it that the previous functions worked for either xarray or Numpy objects, but eq_hist only works with Numpy, so you made it always use Numpy?  That's fine; just curious...